portal/templates/profile_macros.html

```diff
@@ -2050,9 +2046,11 @@ <h4 class="modal-title">{{ _("Enter questionnaire manually on patient's behalf")
 
             var method = $("input[name='entryMethod']:checked").val();
             var completionDate = $("#qCompletionDate").val();
+            var linkUrl = "{{url_for('assessment_engine_api.present_needed')}}" + "?subject_id={{person.id}}";
```

Comment:
Would this be better?
```
{{url_for('assessment_engine_api.present_needed', subject_id=person.id)}}
```

Reply:
@ivan, ivan, thank u, should be fixed now.

Reply:
When you type @ivan anywhere into GitHub, the wrong ivan gets an email